target/linux/ramips/dts/mt7621_asus_rt-ac57u_ac1200gu.dtsi

```diff
@@ -0,0 +1,186 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	aliases {
+		label-mac-device = &wan;
+		led-boot = &led_power;
+		led-failsafe = &led_power;
+		led-running = &led_power;
+		led-upgrade = &led_power;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_power: power {
+			label = "blue:power";
+			gpios = <&gpio 48 GPIO_ACTIVE_LOW>;
+		};
+
+		usb {
```

Comment:
The "label" properties have been deprecated upstream by a combination of [color + function + func-enum properties](https://www.kernel.org/doc/Documentation/devicetree/bindings/leds/common.yaml).

Note: we still need the label property for nodes which are assigned a led-* alias in the aliases node above.

A list of the possible color + function is available in:
https://elixir.bootlin.com/linux/v5.10.87/source/include/dt-bindings/leds/common.h#L24